hub/api/tests/test_api.py

```diff
@@ -964,15 +964,18 @@ def test_compressions_list():
         "bmp",
         "dcm",
         "dib",
+        "eps",
         "flac",
         "gif",
         "ico",
+        "im",
         "jpeg",
         "jpeg2000",
         "lz4",
         "mkv",
         "mp3",
         "mp4",
+        "msp",
```

Comment:
shouldn't mpo and fli be in this list?

Reply:
Mpo was giving error

Reply:
error is fixed?

Reply:
Yes compression_list is not giving errors now